tensorboard/webapp/widgets/linked_time_fob/types.ts

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Just a note that I intend to create types.ts here too. #5602 
For those types used only in linked_time_fob, I'm thinking about put them as "internal_types.ts"